src/pages/study/create/index.tsx

```diff
@@ -50,20 +50,22 @@ const CreateStudy = () => {
   const handleCreateStudy = () => {
     const formData = new FormData();
     const { head_image, ...newStudy } = handleSubmitInput();
-    const emptyList = errorMessage;
+    const errorList = errorMessage;
 
     if (head_image) formData.append('head_image', head_image[0]);
 
     for (const key in newStudy) {
-      if (key !== 'tags') emptyList[key] = validateStudy(newStudy[key]);
-      if (!emptyList[key]) formData.append(key, newStudy[key]);
+      if (key !== 'tags') errorList[key] = validateStudy(newStudy[key]);
+      if (!errorList[key]) formData.append(key, newStudy[key]);
     }
 
-    if (Object.values(emptyList).join('').length) {
-      setErrorMessage({ ...emptyList });
+    if (Object.values(errorList).join('').length) {
+      setErrorMessage({ ...errorList });
       showToast({
         type: 'fail',
-        message: '필수 항목을 모두 입력해주세요.',
+        message: errorList.post_content.includes('script')
```

Comment:
이 부분도 동일하게
```suggestion
        message: validateStudy<string>(newStudy.post_content)
```
처럼 처리하면 되지 않을까요?

Reply:
왜냐하면 validate 함수에서는 <script> </script>가 모두 포함되어야 처리되는데 이도 동일시하게 해주어야 할 것 같아서요!

Reply:
validate 함수가 post_content 항목에 대해 script 태그 포함 여부랑 html 태그를 제외한 문자열의 길이를 같이 판단하고 있어서 `.includes('script')`는 똑같이 붙여야 할 것 같습니다 ㅠㅠ

흠 아니면 차라리 script 태그가 있는지만 확인하는 검증 함수를 따로 작성해볼까요?

Reply:
테스트 해보니 script 태그만 있을 시에 script 경고메세지 잘 나오는 것 같은데요?!

Reply:
아 제가 착각했던 게 post_content.includes 인줄 알았어요..ㅋㅋ
두 번 검증할 필요가 없네요 좋습니다~~